src/task2_1.py

```diff
@@ -0,0 +1,9 @@
+firstname = 'Kate'
+
+lastname = 'Tishevich'
+
+age = '22'
+
+result = ("Хей,я" + " " + firstname + " " + lastname + "," + "мне" + " " + age + " " + "года")
```

Comment:
форматирование сделай пожалуйста через f".."
```suggestion
result = f"Хей,я" + " " + firstname + " " + lastname + "," + "мне" + " " + age + " " + "года"
```

Reply:
@katherinetishevich ?)

Reply:
f"Хей,я {firstname} {lastname}"

Reply:
https://github.com/tamkovich/tms-python-spring/pull/4/commits/a800a47b734b01aaad6e9e00f64eaf8ffd96c674